src/cy-reporter.js

```diff
@@ -36,6 +36,16 @@ var resultsFolder;
  * @returns {testcase record object}
  */
 
+function writeResults(resultsFilePath, xml) {
+  var folder = path.dirname(resultsFilePath);
+  if (!fs.existsSync(folder)){
+    fs.mkdirSync(folder, { recursive: true}, (err) => {
+      if (err) throw err;
+    });
```

Comment:
I might be wrong, but `mkdirSync` does not take a "callback"!
Besides that, what is the point of re-throwing errors?